Makefile

```
@@ -225,8 +229,9 @@ create-kind-multicluster:
 	scripts/setup-kind-multicluster.sh --clusters 2 --kind-worker-nodes 4
```

We'll want to update this script to use the new variable...

```suggestion
	scripts/setup-kind-multicluster.sh --clusters $(NUM_CLUSTERS) --kind-worker-nodes 4
```

I think should do the trick